system-test/bigtable.js

```diff
@@ -372,6 +372,40 @@ describe('Bigtable', function() {
         done
       );
     });
+
+    it('should generate consistency token', done => {
+      TABLE.generateConsistencyToken(function(err, token) {
+        assert.notStrictEqual(token, undefined);
+        assert.strictEqual(typeof token, 'string');
+        done();
+      });
+    });
+
+    it('should return error for checkConsistency of invalid token', done => {
+      TABLE.checkConsistency('dummy-token', function(err, res) {
+        assert.notStrictEqual(err, undefined);
```

Comment:
We don't have to worry about testing the `res` argument, but we should do some type of checking that `err.message` is the message we expect.